events_types.go

```diff
@@ -214,11 +214,19 @@ type EventCrashType struct {
 }
 
 func (ev *EventCrashType) String() string {
-	return fmt.Sprintf("container '%v' has crashed", ev.TypeData.ContainerType)
+	msg := fmt.Sprintf("container '%v' has crashed", ev.TypeData.ContainerType)
+
+	if ev.TypeData.CrashLogs != "" {
+		dashboard_url := "https://my.scalingo.com/apps/" + ev.AppName + "/events/" + ev.ID
```

Comment:
Should'nt we add the dashboard_url to the EventCrash type?
I'm not really a fan of the hard link between go-scalingo and the dashboard. This will not work on staging and dev environments. And if we will change the dashboard url, nobody will think of looking in go-scalingo to see if there is a link here?